### PR TITLE
Added another pattern for Ternary (CONDITION ? OBJECT : OBJECT)

### PR DESCRIPTION
**Pull request by DeltaRays**
### Description

I just added that new pattern and a few examples as after talking with various people they too thought it would've been a good thing to add (I know that skript is supposed to be mostly English but that's why I feel like having symbols separating various English expressions would make even more sense (it's also a clearer separation)
---
**Target Minecraft Versions:**    any 
**Requirements:**   none  
**Related Issues:**   none

### CODE REVIEW

File: src/main/java/ch/njol/skript/expressions/ExprTernary.java [hunk 1]
```diff
@@ -41,16 +41,16 @@
 
 @Name("Ternary")
 @Description("A shorthand expression for returning something based on a condition.")
-@Examples({"set {points} to 500 if {admin::%player's uuid%} is set else 100"})
+@Examples({"set {points} to 500 if {admin::%player's uuid%} is set else 100", "set {points} to (player's name = \"Notch\") ? 500 : 100"})
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/expressions/ExprTernary.java:44]: Each example should be on its own line

File: src/main/java/ch/njol/skript/expressions/ExprTernary.java [hunk 2]
```diff
@@ -41,16 +41,16 @@
...
 @Since("2.2-dev36")
 @SuppressWarnings("null")
 public class ExprTernary<T> extends SimpleExpression<T> {
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/expressions/ExprTernary.java:47]: Preferably (or at least based on the current conventions), there should be no tabs and/or spaces on an empty line. Although it doesn’t matter *too* much, but if you are making other changes you could probably remove them.

File: src/main/java/ch/njol/skript/expressions/ExprTernary.java [hunk 3]
```diff
@@ -41,16 +41,16 @@
...
-
+	
 	static {
 		Skript.registerExpression(ExprTernary.class, Object.class, ExpressionType.COMBINED,
-				"%objects% if <.+>[,] (otherwise|else) %objects%");
+			"%objects% if <.+>[,] (otherwise|else) %objects%", "<.+> ? %objects% : %objects%");
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/expressions/ExprTernary.java:51]: Patterns should be on their own line (for readability IMO)

Comment (DeltaRays) [reply src/main/java/ch/njol/skript/expressions/ExprTernary.java:51]: Sorry, how would I acccept your changes?

Comment (APickledWalrus) [reply src/main/java/ch/njol/skript/expressions/ExprTernary.java:51]: Oh, I didn’t actually propose any changes for you to accept. You can just edit the file and do what I suggested though, and mark the conversation as resolved after you commit.

Comment (DeltaRays) [reply src/main/java/ch/njol/skript/expressions/ExprTernary.java:51]: Okay, thanks

**Comment by DeltaRays** [comment]
Done, but now it tells me you're still requesting a change, do I have to wait until you cancel your request or something?

**Comment by DeltaRays** [comment]
👍 okay

**Pull request closed without merging**